rules/technical-overview/rule.md

```diff
@@ -0,0 +1,47 @@
+---
+type: rule
+title: Rules to having a good technical overview
```

Comment:
Rule titles are written in this format
`Do you ... ?`

```suggestion
title: Do you have a good technical overview?
```

Its also good to add a prefix 
e.g. Twitter - Do you know to thank others for each reference to you?
https://www.ssw.com.au/rules/thank-others-for-each-reference-to-you